R/system.R

```diff
@@ -2993,6 +3007,35 @@ searchAndReadExcelFiles <- function(folder, forPreview = FALSE, pattern = "", sh
 
 }
 
+#'API that imports multiple same structure Excel sheets in the Excel Book and merge them to a single data frame
+#'@export
+read_excel_file_multi_sheets <- function(file, forPrevew = FALSE, sheets = c(1), col_names = TRUE, col_types = NULL, na = "", skip = 0, trim_ws = TRUE, n_max = Inf, use_readxl = NULL, detectDates = FALSE, skipEmptyRows = FALSE, skipEmptyCols = FALSE, check.names = FALSE, tzone = NULL, convertDataTypeToChar = TRUE, ...) {
```

Comment:
Add comment on the default of sheets argument.

Reply:
Added a comment for the "sheets" argument default value.